lib/langchain/agent/chain_of_thought_agent/chain_of_thought_agent.rb

```diff
@@ -81,12 +81,18 @@ def run(question:)
     # @param tools [Array] Tools to use
     # @return [String] Prompt
     def create_prompt(question:, tools:)
+      tool_list = tools.map do |tool|
```

Comment:
If we had the `.name` method this would just become `tool_list = tools.map(&:name)`

Reply:
I added a NAME constant to each tool.